### PR TITLE
clean up authorization references

### PR DESCRIPTION
**Pull request by christinaausley**
## Description

Closes https://github.com/camunda/camunda-docs/issues/2098.

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Review by akeller — CHANGES_REQUESTED** [review]
I see changes to the terms, which look really great!

I'm still missing standardizing on whether we have a devoted page or include it as a section in the overview:

* https://docs.camunda.io/docs/apis-tools/tasklist-api-rest/tasklist-api-rest-authentication/
* https://docs.camunda.io/optimize/apis-tools/optimize-api/optimize-api-authorization/

I could make an argument either way, but we should be consistent across our APIs.

**Comment by akeller** [comment]
@christinaausley do you need any help from me here?

**Comment by christinaausley** [comment]
> @christinaausley do you need any help from me here?

Woah. We both looked at this at the same time! I've built out authentication sections for each API.

- Can we leave it to the sidebar sync upon release to update this in the Optimize sidebar?
- Console needs its own folder to add an authentication section, which means redirects and such. I'm going to open a separate issue for this.

**Comment by christinaausley** [comment]
@akeller Can you have a look at my redirects? Then I think this should be good to go.

File: versioned_docs/version-8.2/apis-tools/working-with-apis-tools.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ Camunda 8 components have APIs to enable polyglot developers to work with in the
 type:"link", href:"/docs/next/apis-tools/operate-api/overview/", label: "Operate API (REST)", docId:"apis-tools/operate-api/operate-api-overview"
 },
 {
-type:"link", href:"/optimize/next/apis-tools/optimize-api/optimize-api-authorization/", label: "Optimize API (REST)"
+type:"link", href:"/optimize/apis-tools/optimize-api/optimize-api-authorization/", label: "Optimize API (REST)"
```
Comment (pepopowitz) [inline versioned_docs/version-8.2/apis-tools/working-with-apis-tools.md:78]: Should this be changed from `..../optimize-api-authorization` to `..../optimize-api-authentication`?

File: versioned_docs/version-8.3/apis-tools/working-with-apis-tools.md [hunk 2]
```diff
@@ -60,7 +60,7 @@ Camunda 8 components have APIs to enable polyglot developers to work with in the
 type:"link", href:"/docs/next/apis-tools/operate-api/overview/", label: "Operate API (REST)", docId:"apis-tools/operate-api/operate-api-overview"
 },
 {
-type:"link", href:"/optimize/next/apis-tools/optimize-api/optimize-api-authorization/", label: "Optimize API (REST)"
+type:"link", href:"/optimize/apis-tools/optimize-api/optimize-api-authorization/", label: "Optimize API (REST)"
```
Comment (pepopowitz) [inline versioned_docs/version-8.3/apis-tools/working-with-apis-tools.md:63]: Should this be changed from `..../optimize-api-authorization` to `..../optimize-api-authentication`?

**Review by pepopowitz — COMMENTED** [review]
I don't think this is the reason link-checking is failing, but I noticed these while investigating.

File: static/.htaccess [hunk 3]
```diff
@@ -10,6 +10,10 @@ Options -Indexes -MultiViews
 RewriteRule ^docs/8.3/(.*)$ /docs/$1 [R=302,L]
 RewriteRule ^optimize/3.11.0/(.*)$ /optimize/$1 [R=302,L]
 
+# Authorization to authentication
+RewriteRule ^optimize/next/apis-tools/optimize-api/optimize-api-authorization/(.*)$ /optimize/next/apis-tools/optimize-api/optimize-api-authentication/$1 [R=301,L]
+RewriteRule ^optimize/apis-tools/optimize-api/optimize-api-authorization/(.*)$ /optimize/apis-tools/optimize-api/optimize-api-authentication/$1 [R=301,L]
```
Comment (pepopowitz) [inline static/.htaccess:15]: @christinaausley I think this rewrite rule is causing the link-checking failure!

This file was moved in the `next` version (`docs/` folder), but it wasn't moved in the current version (`version-8.3/`). 

The server is redirecting this URL from the sitemap away from a page that does exist (current version/authorization) to a page that _doesn't_ exist (current version/authentication).

File: docs/apis-tools/operate-api/authentication.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
+---
+id: operate-api-authentication
+title: Authentication
+slug: /apis-tools/operate-api/authentication
```
Comment (akeller) [inline docs/apis-tools/operate-api/authentication.md:4]: Does this need a slug?

File: static/.htaccess (hunk 3 above)
Comment (akeller) [reply static/.htaccess:15]: ☝️ I'm too slow 😆 I came here to say most changes are only applied to `/next/`. But I was too focused on debugging redirects. Thanks @pepopowitz!

File: versioned_docs/version-8.3/apis-tools/working-with-apis-tools.md (hunk 2 above)
Comment (pepopowitz) [reply versioned_docs/version-8.3/apis-tools/working-with-apis-tools.md:63]: No, it shouldn't! I made this comment before noticing the URL was only moved in `next`.

File: versioned_docs/version-8.2/apis-tools/working-with-apis-tools.md (hunk 1 above)
Comment (pepopowitz) [reply versioned_docs/version-8.2/apis-tools/working-with-apis-tools.md:78]: No it shouldn't! I made this comment before noticing the URL was only moved in `next`.

File: static/.htaccess (hunk 3 above)
Comment (christinaausley) [reply static/.htaccess:15]: @akeller Are you fine with these changes going out with the minor release? If so, I'll just remove the redirect 👍

Comment (akeller) [reply static/.htaccess:15]: Sure!

**Review by akeller — APPROVED** [review]

**Pull request merged**